repair-service/model/src/main/java/com/epam/Client.java

```diff
@@ -0,0 +1,169 @@
+package com.epam;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * The type Client.
```

Comment:
`Class represents entity with client information of repairing service.`
JavaDoc should be informative. I see that this is a `class Client`